dbt_subprojects/solana/models/tokens/solana/tokens_solana_transfers.sql

```diff
@@ -4,25 +4,28 @@
         alias = 'transfers',
         materialized = 'incremental',
         file_format = 'delta',
-        incremental_strategy = 'merge',
+        incremental_strategy = 'delete+insert',
         partition_by = ['block_date'],
-        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_date')],
```

Comment:
@0xBoxer apparently the `DBT_INTERNAL_DEST` prefix is specific to the `merge` strategy. Just using `block_date` should work :pray:

Reply:
aha, thanks 🙏